php/php.editor/test/unit/data/testfiles/completion/lib/nb1855/nb1855.php.testCompleteAccessPrefixInClassConst_01.completion

```diff
@@ -6,8 +6,8 @@ CLASS      ParentClass                     [PUBLIC]   nb1855.php
 CONSTANT   PRIVATE_CONSTANT "private cons  [PRIVATE]  Class1855
 CONSTANT   PROTECTED_CONSTANT "protected   [PROTECTE  Class1855
 CONSTANT   PROTECTED_PARENT_CONSTANT "pro  [PROTECTE  ParentClass
-CONSTANT   PUBLIC_CONSTANT "pubic constan  [PUBLIC]   Class1855
-CONSTANT   PUBLIC_PARENT_CONSTANT "pubic   [PUBLIC]   ParentClass
+CONSTANT   PUBLIC_CONSTANT "public consta  [PUBLIC]   Class1855
+CONSTANT   PUBLIC_PARENT_CONSTANT "public  [PUBLIC]   ParentClass
```

Comment:
So you're changing a ".completion" test file. 
Questions:
1) Can you explain how these completion test work?
2) Does this module pass an `ant test` after this change?

Reply:
What are the answers to these two questions?

Reply:
omg how long test is
1. no
2. too long execution to test. I run it with ant test from php/php.editor